test/test_extended_models.py

```diff
@@ -201,19 +202,31 @@ def test_schema_meta_validation(model_fn):
                 unquantized_w = w.meta.get("unquantized")
                 if unquantized_w is not None and w.meta.get("num_params") != unquantized_w.meta.get("num_params"):
                     incorrect_params.append(w)
+
+                # the methodology for quantized ops count doesn't work as well  so we take unquantized FLOPs instead
+                calculated_ops = get_ops("models", model_name, unquantized_w)
             else:
                 if w.meta.get("num_params") != sum(p.numel() for p in model_fn(weights=w).parameters()):
                     incorrect_params.append(w)
+
+                calculated_ops = get_ops(module_name, model_name, w)
```

Comment:
we need to review this logic. This way we initialize the models multiple times. Once on the `model_fn` call above and once within `get_ops()`. What we can do is initialize the model once and then use it in both cases.